### ISSUE TITLE
Tracking stops for sessions with a terminal loaded in a window

### ISSUE THREAD

**Issue by tssm** [dominickng/fzf-session.vim#9]
On Neovim 0.4.3 when session with a terminal in a window is open I get

```
Error detected while processing function 6[30..<SNR>28_callbak:
line 21:
Vim(normal):Can't re-enter normal mode from terminal mode
```

After that no changes in the session get recorded so I need to overwrite it (and make sure there's no terminal buffer in any window).

It doesn't happen with other plug-ins like Startify. I tried to fix it myself but don't get what the error is referring to

**Comment by dominickng** [comment]
Hmm, interesting. I don't use neovim myself, but it looks like it has a [terminal mode](https://neovim.io/doc/user/nvim_terminal_emulator.html) that might be the source of this error.

Do you have any other bindings that might be triggering terminal mode that could be interacting with fzf?

**Comment by tssm** [comment]
Yes, the terminal is the source of the issue: it only occurs when I close a session with a terminal buffer loaded in a window but if I close all the windows with a terminal before I close the session there's no issue.

Apparently the issue is in the load step: I created a session with a terminal on a window using FZF Session, but then I loaded it manually (with `:source`) and it works. I'll try to track it after work and see if I can send a PR
